php/media/class-video.php

```diff
@@ -191,9 +191,21 @@ public function enqueue_block_assets() {
 	 * Register assets for the player for preview.
 	 */
 	public function admin_enqueue_scripts() {
-		wp_register_style( 'cld-player', 'https://unpkg.com/cloudinary-video-player@' . self::PLAYER_VER . '/dist/cld-video-player.min.css', null, self::PLAYER_VER );
-		wp_register_script( 'cld-core', 'https://unpkg.com/cloudinary-core@' . self::CORE_VER . '/cloudinary-core-shrinkwrap.min.js', null, self::CORE_VER, true );
-		wp_register_script( 'cld-player', 'https://unpkg.com/cloudinary-video-player@' . self::PLAYER_VER . '/dist/cld-video-player.min.js', array( 'cld-core' ), self::PLAYER_VER, true );
+		$current_screen    = get_current_screen();
+		$requiring_screens = array(
```

Comment:
@pereirinha - It would be nice to have the required screens on a filter to allow other plugins to extend it.